### ISSUE TITLE
Duplicate dialog

### ISSUE THREAD

**Issue by krisgesling** [MycroftAI/skill-homeassistant#92]
**Describe the bug**
User on the Forums is reporting that they get dialog duplicated as output:
> “Successfully set thermostat to 70 none, successfully set thermostat to 70 none”

It's also reporting "None" for presumably the unit of measure.

**To Reproduce**
Full details here:
https://community.mycroft.ai/t/mycroft-speaks-twice-with-homeassistant-confirmations/11428/2

**Expected behavior**
Single dialog reported back.

**Comment by Tony763** [comment]
Returning `None` should not occur with latest version. We will see if sparkyvision give us some more feedback.


I only found problem within `__init__.py` line 576, when HA do not return current temp. Intent get value of sensor:
```
File "/srv/mycroft-core/mycroft/skills/mycroft_skill/event_container.py", line 73, in wrapper
    handler(message)
  File "/opt/mycroft/skills/skill-homeassistant.mycroftai/__init__.py", line 215, in handle_sensor_intent
    self._handle_sensor(message)
  File "/opt/mycroft/skills/skill-homeassistant.mycroftai/__init__.py", line 576, in _handle_sensor
    current_temp = nice_number((float(attributes['current_temperature'])))
TypeError: float() argument must be a string or a number, not 'NoneType'
```
![image](https://user-images.githubusercontent.com/5598750/139223778-7a5d083b-520e-473e-b2a8-dc1242b19bb2.png)

We will have to divide response dialogs.

**Comment by pfefferle** [comment]
> We will have to divide response dialogs.

Makes sense! This was also a feedback of @krisgesling https://github.com/MycroftAI/skill-homeassistant/pull/82#discussion_r728653195

**Comment by Tony763** [comment]
I will prepare it tonight, I will just need translations to DE.

**Comment by pfefferle** [comment]
I can help with german translations.

**Comment by Tony763** [comment]
@pfefferle PR ready, could You add translations and test it out? [Tony763:feature/issue92](https://github.com/Tony763/skill-homeassistant/tree/feature/issue92)

**Comment by pfefferle** [comment]
@Tony763 sure, will do it later today.

**Comment by Tony763** [comment]
Wow, this one ended a half way. I missed Your PR with translations. 

I will try to update changes with actual code base.
